indexing-service/src/main/java/io/druid/indexing/overlord/supervisor/SupervisorManager.java

```diff
@@ -181,7 +182,13 @@ private boolean createAndStartSupervisorInternal(SupervisorSpec spec, boolean pe
     supervisor.start(); // try starting the supervisor first so we don't persist a bad spec
```

Comment:
I notice this comment here.  I wonder if the supervisor shouldn't have a `.verify()` call on it to make sure the spec is legit, then persist to DB then `start()`?

Or, maybe just store in the DB first, `start()` and if `start()` doesn't work, fail out and remove from the DB again?